R/track.R

```diff
@@ -0,0 +1,403 @@
+#' Compute track statistics
+#'
+#' @param population, single cell data
+#' @param grouping_variable, column name storing the track label
+#' @return track
+#' @importFrom magrittr %>%
+#' @examples 
+#'  data <- tibble::data_frame(
+#'    Metadata_timePoint = c(1:5),
+#'    Location_Center_X = c(1, 2, 3, 4, 5),
+#'    Location_Center_Y = c(1, 1, 1, 1, 1),
+#'    TrackObjects_Label = c(rep(1, 5))
+#'  )
+#'  data <- dplyr::group_by_(data,'TrackObjects_Label')
+#'  tracks <- track(data,'TrackObjects_Label')
+#' @export
+track <- function(population, grouping_variable) {
+  # process `population`, which is the data you get from CellProfiler
+  tracks <- displace(population, grouping_variable)
+  
+  features <- list(
+    angle(tracks),
+    chemotacticIndex(tracks),
```

Comment:
I recommend we use http://r-pkgs.had.co.nz/r.html as a starting point for the style guide